apps/posts/test/unit/hooks/usePostNewsletterStats.test.tsx

```diff
@@ -1,60 +1,60 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import {beforeEach, describe, expect, it, vi} from 'vitest';
-import {createTestWrapper, mockApiHook, setupUniversalMocks} from '../../utils/test-helpers';
+import {beforeEach, describe, expect, it} from 'vitest';
+import {createTestWrapper, mockData, mockServer} from '../../utils/msw-helpers';
```

Comment:
as I was looking through this test, I started wondering about `usePostNewsletterStats.ts`.

And saw these functions, which might be decoupled out of hook to test easier as unit tests, since that hook seems big:

* `const stats = useMemo(() ...` 
* feedbackStats
* newsletterStatsResponse
* averages

just a thought